typesense-java/src/test/java/org/typesense/model/ApiTest.java

```diff
@@ -46,7 +48,8 @@ public void testPost(){
     public void testGet() {
         Api api = new Api(configuration);
 
-        Document d = api.get("/collections/intbooks/documents/Odyssey",Document.class);
-        System.out.println(d);
+        CollectionSchema c = api.get("/collections/intbooks",CollectionSchema.class);
```

Comment:
This is just a minor comment, I've taken to using the form for tests as: shouldDoX versus testX (and the @Test annotation) as it's slightly more descriptive and potentially helps frame the intent of the test. Some tools will parse that form I think to produce documentation like "Should Do Foo to Bar". But just a personal preference.

Reply:
Okay! Seems fair! Let's stick with your suggestion